package.json

```diff
@@ -84,6 +84,9 @@
     "vite-plugin-istanbul": "4.0.1"
   },
   "resolutions": {
+    "@graasp/ui": "github:graasp/graasp-ui#322-fix-app-iframe-loading-forever",
+    "@graasp/sdk": "0.10.1",
+    "immutable": "4.3.0",
```

Comment:
Do we need these resolutions?

Reply:
I am trying to make it so that we do not need them, but it takes some effort to try to make all packages converge and make them less dependent on each other.